x-pack/plugins/snapshot_restore/server/routes/api/repositories.ts

```diff
@@ -156,3 +160,27 @@ export const updateHandler: RouterRouteHandler = async (req, callWithRequest) =>
     verify: false,
   });
 };
+
+export const deleteHandler: RouterRouteHandler = async (req, callWithRequest) => {
+  const { names } = req.params;
+  const repositoryNames = names.split(',');
+  const response: { success: string[]; error: any[] } = {
+    success: [],
+    error: [],
+  };
+
+  await Promise.all([
+    ...repositoryNames.map(name => {
```

Comment:
Minor nit: `map` returns a new array, so I think you can remove the wrapping array literal and remove the destructuring. Same on line 43.